Sample.java

```diff
@@ -1,7 +1,51 @@
-// Time Complexity :
+// Time Complexity :0(1)
 // Space Complexity :
```

Comment:
Include time and space complexities as per your understandings, we'll check afterwards